controllers/hyperconverged/hyperconverged_controller.go

```diff
@@ -80,13 +80,24 @@ const (
 	hcoVersionName    = "operator"
 	secondaryCRPrefix = "hco-controlled-cr-"
 	apiServerCRPrefix = "api-server-cr-"
+	configMapCRPrefix = "cm-policy-cr-"
 
 	// These group are no longer supported. Use these constants to remove unused resources
 	v2vGroup = "v2v.kubevirt.io"
 
 	requestedStatusKey = "requested status"
 )
 
+// triggeredBy informs which resource has triggered the Reconcile loop
+type triggeredBy int8
+
+const (
+	triggeredByHco triggeredBy = iota
+	triggeredByApi
+	triggeredByCm
```

Comment:
Nit:

HCO, API and CM are acronyms. Please rename:
 ```suggestion
	triggeredByHCO triggeredBy = iota
	triggeredByAPI
	triggeredByCM
```

Reply:
Addressed.